examples/datadog/src/main.rs

```diff
@@ -17,7 +17,7 @@ fn bar() {
 }
 
 fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
-    let (tracer, _uninstall) = opentelemetry_contrib::datadog::new_pipeline()
+    let (tracer, _uninstall) = opentelemetry_contrib::trace::exporter::datadog::new_pipeline()
```

Comment:
small nit: may want to `use` this as above? calling a function 4 namespaces deep currently 😂

Reply:
Yeah definitely should have imported them at the top